pkg/cmd/cmd.go

```diff
@@ -105,9 +112,111 @@ func handleResponse(body []byte, contentType string, statusCode int, cmd *cobra.
 
 			return fmt.Errorf(jsonErr)
 		default:
-			return fmt.Errorf("request failed with status code %d, error: %s\n", statusCode, string(body))
```

Comment:
Intended drop of the newline at end? I think also maybe this is accidentally `%s` format verb but probably should be `%d` as the one below.